internal/nlutil/nlutil_linux.go

```diff
@@ -0,0 +1,140 @@
+// Package nlutil wraps vishvandanda/netlink functions that may return EINTR.
+//
+// The wrapped functions currently return EINTR when NLM_F_DUMP_INTR is, meaning
+// something changed during the dump so results may be incomplete or inconsistent.
+//
+// At present, the possibly incomplete/inconsistent results are not returned
+// by netlink functions along with the EINTR. So, it's not possible to do
+// anything but retry.
+//
+// These wrappers will make maxAttempts before returning the EINTR.
+package nlutil
+
+import (
+	"context"
+	"errors"
+
+	"github.com/containerd/log"
+	"github.com/vishvananda/netlink"
+	"golang.org/x/sys/unix"
+)
+
+const maxAttempts = 5
```

Comment:
Can you add a comment "why 5"?

Reply:
Yes - done ... it's an arbitrary limit, just a small number that's more than one!